### PR TITLE
Typescript and tests improvements

### PR DESCRIPTION
**Pull request by robinpiets**
As discussed we would like to see some improvements in using `otherProps`. Also, I've implemented the feedback from @afjlambert on the TS PR.  I've also improved the test coverage by skipping some code which is no useful or too hard to test.

Please make sure:
- [x] to add a label
- [x] to add one or more reviewers
- [x] you followed our checklist for [functional testing](https://github.com/Amsterdam/vergunningcheck/blob/master/TESTING.md)
- [x] you added the necessary documentation (either in the markdown files or inline)
- [x] you added the necessary automated tests

### CODE REVIEW

File: apps/client/src/components/Layouts/DefaultLayout.test.js [hunk 1]
```diff
@@ -14,24 +14,33 @@ jest.mock("react-router-dom", () => ({
...
 
-const Wrapper = ({ children }) => {
-  const topicMock = "dakraam-plaatsen";
-  const topicUrlMock = `/${topicMock}`;
-  const topic = findTopicBySlug(topicMock);
+const topicMock = "dakraam-plaatsen";
+const topicUrlMock = `/${topicMock}`;
+const topic = findTopicBySlug(topicMock);
 
+const Wrapper = ({ children, ...otherProps }) => {
```
Comment (robinpiets) [inline apps/client/src/components/Layouts/DefaultLayout.test.js:21]: 🙈

Comment (svenjens) [reply apps/client/src/components/Layouts/DefaultLayout.test.js:21]: 😂

**Review by svenjens — APPROVED** [review]

File: apps/client/src/components/Layouts/DefaultLayout.test.js (hunk 1 above)
Comment (afjlambert) [reply apps/client/src/components/Layouts/DefaultLayout.test.js:21]: 😲

File: apps/client/src/components/HiddenDebugInfo.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* istanbul ignore file */
```
Comment (afjlambert) [inline apps/client/src/components/HiddenDebugInfo.js:1]: nice

**Review by afjlambert — APPROVED** [review]

**Pull request merged**